TripKit/Classes/model/API/AlertAPIModels.swift

```diff
@@ -61,7 +61,15 @@ extension API {
     public let operators: [String]?
     public let serviceTripIDs: [String]?
     public let stopCodes: [String]?
-    public let routeIDs: [String]?
+//    public let routeIDs: [String]?
```

Comment:
This is the API provided by Fernando,

https://bigbang.skedgo.com/satapp-beta/alerts/transit.json?region=AU_VIC_Melbourne

It doesn't include `routeIDs` field but `routes`. Which one will make it to our public API?